customer-api/src/Controller/StringManipulationController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class StringManipulationController extends AbstractController
+{
+    /**
+     * @Route("/string-manipulate", name="string_manipulate")
+     */
+    public function index(Request $request): Response
+    {
+        $inputString = $request->query->get('input');
+        $targetString = $request->query->get('target');
+        $result = null;
+
+        if ($inputString !== null && $targetString !== null) {
+            $result = $this->validateStringFromInput($inputString, $targetString) ? 'True' : 'False';
+        }
+
+        return $this->render('string_manipulation/index.html.twig', [
+            'result' => $result,
+        ]);
+    }
+
+
+    public function validateStringFromInput(string $input, string $target): void {
+        $t = intval(trim($target));
```

Comment:
use meaningful names, otherwise it adds confusion. in this case confusion is implemented by first using '$t'  as a clean/cast to int -> then in line 33 it's used as a loop bound. but then again it's reset in line 35 within the loop. Let's keep stackoverflow a website we visit, not a feature